Bluepill-cli/Bluepill-cli/Simulator/SimulatorHelper.m

```diff
@@ -128,12 +128,14 @@ + (NSString *)testEnvironmentWithConfiguration:(BPConfiguration *)config {
         BPXCTestFile *xctTestFile = [BPXCTestFile BPXCTestFileFromExecutable:executable isUITestFile:(config.testRunnerAppPath == nil) withError:&error];
         NSAssert(xctTestFile != nil, @"Failed to load testcases from %@", [error localizedDescription]);
         NSMutableSet *testsToSkip = [[NSMutableSet alloc] initWithArray:xctTestFile.allTestCases];
-        NSSet *testsToRun = [[NSSet alloc] initWithArray:config.testCasesToRun];
+        NSSet *testsToRun = [[NSSet alloc] initWithArray:[xctTestFile expandTests:config.testCasesToRun]];
```

Comment:
I don't follow. Can you elaborate? It seems that the purpose of `BPXCTestFileFromExecutable: isUITestFile:withError:` is only to create the `xctTestFile` so I'm not sure what you have in mind. Let me know, thanks.